src/components/atoms/Accordion.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+export default {
+    data() {
+        return {
+            isOpen: false,
+        };
+    },
+    methods: {
+        toggleAccordion() {
+            this.isOpen = !this.isOpen;
+        },
+    },
+};
+</script>
+
+<template>
+    <div>
+        <button
+            @click="toggleAccordion()"
+            class="flex items-center justify-between w-full p-2 text-sm font-semibold text-left text-gray-400 bg-gray-800 rounded-[4px] hover:bg-gray-700 focus:outline-none focus:shadow-outline"
```

Comment:
Don't use any colors not from the custom theme to avoid them clashing if they change

Reply:
I'll need to do this in a few places in the app. I'll do a pass after work today and migrate everything over to the theme colors!